Vagrantfile

```diff
@@ -357,8 +391,18 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
       if File.exist?(cfg)
         kHost.vm.provision :file, :source => "#{cfg}", :destination => "/tmp/vagrantfile-user-data"
+        if enable_proxy
+          kHost.vm.provision :shell, :privileged => true,
+          inline: <<-EOF
+          sed -i "s|__PROXY_LINE__||g" /tmp/vagrantfile-user-data
```

Comment:
Yes. I've tried running cluster at home (without corporate proxy) and everything works as expected.